docs/UserGuide.md

```diff
@@ -248,6 +248,24 @@ Exits the program.
 
 Format: `exit`
 
+### Teams related commands
+
+All teams related commands start with the prefix: `team`
+
+#### Exporting team details: `export`
+
+Copies to your clipboard the details of all the members in a particular team, so that you have an easy time signing up for hackathons!
+
+Format: `INDEX export`
```

Comment:
Actually I think we can have the full command here, otherwise it might be confusing for some users
```suggestion
Format: `team INDEX export`
```